Containerfile.cli-artifacts

```diff
@@ -41,31 +48,23 @@ RUN mkdir -p ${HOME} && \
     chmod -R a+rwx /var/log/nginx && \
     chown -R ${USER_UID}:0 /var/log/nginx && \
     chmod -R a+rwx /var/run && \
-    chown -R ${USER_UID}:0 /var/run
+    chown -R ${USER_UID}:0 /var/run && \
+    chmod +x ${ENTRYPOINT_PATH}
 
 USER ${USER_UID}
 
 WORKDIR ${HOME}/src
 
-COPY --from=builder /app/bin/clis/archives/ ./
+COPY --from=builder /app/bin/clis/gh-archives/ ${HOME}/src/gh-archives/
+
+USER root
+RUN chmod a+rwx ${HOME}/src/gh-archives/ -R
+USER ${USER_UID}
 
 LABEL io.k8s.display-name="Flight Control CLI multiarch artifacts with server" \
       io.k8s.description="Flight Control is a service for declarative management of fleets of edge devices and their workloads." \
       io.openshift.tags="flightctl,cli-artifacts"
 
+EXPOSE 8090
 
-EXPOSE 8080
-
-ENTRYPOINT if [[ -d "/proc/sys/net/ipv4" && -d "/proc/sys/net/ipv6" ]]; \
-    then \
-    nginx -g "daemon off;"; \
-    elif [[ -d "/proc/sys/net/ipv4" ]]; \
-    then \
-    nginx -c /etc/nginx/nginx.conf.ipv4 -g "daemon off;"; \
-    elif [[ -d "/proc/sys/net/ipv6" ]]; \
-    then \
-    nginx -c /etc/nginx/nginx.conf.ipv6 -g "daemon off;"; \
-    else \
-    echo "unable to identify IP configuration"; \
-    exit -1; \
-    fi
+ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
```

Comment:
lets reuse the `ENTRYPOINT_PATH` env var here

Reply:
done